### PR TITLE
Adding the demo script

### PR DESCRIPTION
**Pull request by nikhiljindal**
Adding the demo script so that others can try it out as well.

cc @G-Harmon

### CODE REVIEW

File: demo/demo.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Copyright 2017 The Kubernetes Authors.
```
Comment (G-Harmon) [inline demo/demo.sh]: copyright google?

File: demo/util.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors.
```
Comment (G-Harmon) [inline demo/util.sh]: google.

File: demo/demo.sh (hunk 1 above)
Comment (nikhiljindal) [reply demo/demo.sh]: aah fixed

File: demo/util.sh (hunk 2 above)
Comment (nikhiljindal) [reply demo/util.sh]: fixed

**Comment by nikhiljindal** [comment]
Thanks for catching that @G-Harmon.
Fixed and pushed updated code

**Pull request merged**